firefox-ios/EcosiaTests/Analytics/AnalyticsSpyTests.swift

```diff
@@ -848,7 +848,7 @@ final class AnalyticsSpyTests: XCTestCase {
 
     func testClearWebsitesDataTracksEvent() {
         // Arrange
-        let vc = WebsiteDataManagementViewController()
+        let vc = WebsiteDataManagementViewController(windowUUID: .XCTestDefaultUUID)
```

Comment:
If we change this here too, we will have a small conflict when https://github.com/ecosia/ios-browser/pull/845 gets merged

Reply:
I actually [commited directly on the upgrade branch too](https://github.com/ecosia/ios-browser/commit/26b1763cb37fe31a99e82958d7ed36fc0d0bf467) because it was affecting every PR, just didn't rebase these two, so should be fine.

Not sure why this started to break only now 🤷